db/database.rb

```diff
@@ -68,9 +68,9 @@ class Organization
 
 class Repository
   include DataMapper::Resource
-  property :rname, String, :required => true, :key => true
+  property :rname, String, :required => true, :key => true, :unique_index => false
 
-  belongs_to :organization, :required => false
+  belongs_to :organization, :required => false, :key => true
```

Comment:
Do you know any way that we could fix the migration in this case?. This fix is important for me as we do have two repos named the same under different orgs.